lib/versioning/composer/index.spec.ts

```diff
@@ -148,6 +156,36 @@ describe('semver.getNewValue()', () => {
       })
     ).toEqual('^1.0');
   });
+  it('bumps caret to same', () => {
+    expect(
+      semver.getNewValue({
+        currentValue: '^1.0.0',
+        rangeStrategy: 'bump',
+        fromVersion: '1.0.0',
+        toVersion: '1.3.5',
+      })
+    ).toEqual('^1.0');
+  });
+  it('replaces caret to same', () => {
+    expect(
+      semver.getNewValue({
+        currentValue: '^1.0.0',
+        rangeStrategy: 'replace',
+        fromVersion: '1.0.0',
+        toVersion: '1.3.5',
+      })
+    ).toEqual('^1.0');
+  });
+  it('replaces short caret', () => {
+    expect(
+      semver.getNewValue({
+        currentValue: '^1.0',
+        rangeStrategy: 'replace',
+        fromVersion: '1.0.0',
+        toVersion: '2.3.5',
+      })
+    ).toEqual('^2.0');
```

Comment:
I agree with this expected value, but also I think it doesn't matter too much if Renovate returns `^2.0.0` instead.

Reply:
Yeah, this is a typo on my part.